### PR TITLE
Add Sea Level Expression

### PR DESCRIPTION
**Pull request by Mr-Darth**
### Description
Added an expression to get the sea level of a world.

---
**Target Minecraft Versions:** any
**Requirements:** none
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+
+@Name("Sea Level")
+@Description("Gets the sea level of a world.")
+@Examples("send \"The sea level in your world is %sea level in player's world%\"")
+public class ExprSeaLevel extends SimplePropertyExpression<World, Long> {
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java:32]: You should use `Number` instead of `Long`

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java (hunk 1 above)
Comment (Mr-Darth) [reply src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java]: Done!

File: src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		register(ExprSeaLevel.class, Number.class, "sea level", "worlds");
+	}
+	
+	@Override
+	public Number convert(World world) {
+		return world.getSeaLevel();
+	}
+	
+	@Override
+	public Class<? extends Long> getReturnType() {
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java:46]: Oh, this should extend and return Number too.

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java (hunk 2 above)
Comment (Mr-Darth) [reply src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java]: Oopsie, didn't see that 😅. Should be fixed now!

File: src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
```
Comment (ShaneBeee) [inline src/main/java/ch/njol/skript/expressions/ExprSeaLevel.java:18]: bensku has updated the last line of the license heard which can be found [**HERE**](https://github.com/SkriptLang/Skript/blob/master/licenseheader.txt) to update accordingly

**Review by ShaneBeee — CHANGES_REQUESTED** [review]
one small change, other than that, looks good to me.

**Review by ShaneBeee — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by ShaneBeee** [comment]
Thank you for your contribution

**Pull request merged**